about/contribute-to-docs.md

```diff
@@ -257,12 +257,12 @@ device.
 
 &#10060; "Perform a bare metal installation."
 
-&#128077; "Install TimescaleDB on bare metal."
+&#128077; "Install Timescale on bare metal."
 
 *Bare-metal*
 : Adjective.
 
-&#10060; "Install TimescaleDB on bare-metal."
+&#10060; "Install Timescale on bare-metal."
```

Comment:
```suggestion
&#10060; "Install TimescaleDB on bare-metal."
```
I feel like this is a case where it remains DB.

Reply:
This is in the Word Usage Guide, it's just an example.